### PR TITLE
Add wasm-pack docs

### PR DESCRIPTION
**Pull request by neilweidinger**
Thanks!

### CODE REVIEW

File: src/unofficial.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ Application domains:
 * [A thoughtful introduction to the pest parser](https://pest.rs/book/)
 * [Rust and WebAssembly](https://rustwasm.github.io/docs/book/)
 * [The `wasm-bindgen` Guide](https://rustwasm.github.io/docs/wasm-bindgen/)
+* [The `wasm-pack` Docs](https://rustwasm.github.io/docs/wasm-pack/)
```
Comment (glyn) [inline src/unofficial.md:18]: As I mention in the README, my convention is to use the literal title of the book in question:
```suggestion
* [Hello wasm-pack!](https://rustwasm.github.io/docs/wasm-pack/)
```

**Review by glyn — CHANGES_REQUESTED** [review]
Thanks for this! Please could you change to use the literal title of the book for consistency with the other entries and as described in the README?

**Comment by neilweidinger** [comment]
Fixed in the latest commit, cheers!

**Review by glyn — APPROVED** [review]
Looks good now, thanks.

**Pull request merged**